src/main/java/Wolox/training/services/OpenLibraryService.java

```diff
@@ -0,0 +1,89 @@
+package Wolox.training.services;
+
+import Wolox.training.DAO.BookDAO;
+import org.json.*;
+import org.springframework.boot.jackson.JsonObjectDeserializer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
+
+public class OpenLibraryService {
+
+    private String titleTag = "title";
```

Comment:
when declaring strings like this you should use the static and final modifier.
`private static final string TITLE_TAG = "title";`